lms/templates/calculator/toggle_calculator.html

```diff
@@ -0,0 +1,124 @@
+                  <li class="hint-item" id="hint-howto-constants" tabindex="-1"><p>For constants, indicate multiplication explicitly (example: 5*c).</p></li>
+                  <li class="hint-item" id="hint-howto-maffixes" tabindex="-1"><p>For affixes, type the number and affix without a space (example: 5c).</p></li>
+                  <li class="hint-item" id="hint-howto-functions" tabindex="-1"><p>For functions, type the name of the function, then the expression in parentheses.</p></li>
+                </ul>
+              </li>
+
+              <li class="hint-item" id="hint-list" tabindex="-1">
+                <table class="calculator-input-help-table">
+                  <tbody>
+                    <tr>
+                      <th scope="col">${_("To Use")}</th>
+                      <th scope="col">${_("Type")}</th>
+                      <th scope="col">${_("Examples")}</th>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Numbers")}</th>
+                      <td>Integers<br />
+                        Fractions<br />
+                        Decimals
+                      </td>
+                      <td>2520<br />
+                        2/3<br />
+                        3.14, .98
+                      </td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Operators")}</th>
+                      <td>+ - * / (add, subtract, multiply, divide)<br />
+                        ^ (raise to a power)<br />
+                        _ (add a subscript)<br />
+                        || (parallel resistors)
+                      </td>
+                      <td>x+(2*y)/x-1
+                        x^(n+1)<br />
+                        v_IN+v_OUT<br />
+                        1||2
+                      </td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Greek letters")}</th>
+                      <td>Name of letter</td>
+                      <td>alpha<br />
+                        lambda
+                      </td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Constants")}</th>
+                      <td>c, e, g, i, j, k, pi, q, T</td>
+                      <td>20*c<br />
+                        418*T
+                      </td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Affixes")}</th>
+                      <td>Percent sign (%) and metric affixes (d, c, m, u, n, p, k, M, G, T)</td>
+                      <td>20%<br />
+                        20c<br />
+                        418T
+                      </td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Basic functions")}</th>
+                      <td>abs, exp, fact or factorial, ln, log2, log10, sqrt</td>
```

Comment:
> To clarify: these are the symbols that the calculator recognize so it's a bad idea to leave them in the hands of translators to be messed with.

@sarina , @talbs Yes, I'm totally agree about the symbols. But what about `or` in the `fact or factorial` part? I guess it should be translated.